trunk/SUAVE/Plots/Geometry_Plots/plot_vehicle.py

```diff
@@ -378,108 +378,76 @@ def plot_propeller_geometry(axes,prop,propulsor,propulsor_name):
         rot    = prop.rotation[n_p] 
         a_o    = 0
         flip_1 = (np.pi/2)  
-        flip_2 = (np.pi/2) 
+        flip_2 = (np.pi/2)  
+        
+        MCA_2d = np.repeat(np.atleast_2d(MCA).T,dim,axis=1)
+        b_2d   = np.repeat(np.atleast_2d(b).T  ,dim,axis=1)
+        t_2d   = np.repeat(np.atleast_2d(t).T  ,dim,axis=1)
+        r_2d   = np.repeat(np.atleast_2d(r).T  ,dim,axis=1)
         
         for i in range(num_B):   
             # get airfoil coordinate geometry     
             airfoil_data = import_airfoil_geometry(a_sec,npoints=n_points)   
```

Comment:
This is awesome!! It's also much less code!

Reply:
Thanks!